### PR TITLE
Implement Ser Gregor Clegane.

### PR DESCRIPTION
**Pull request by ystros**

### CODE REVIEW

File: server/game/cards/characters/02/sergregorclegane.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+class SerGregorClegane extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onPillage: (event, challenge, card, discarded) => card === this && discarded.getType() === 'character'
+            },
+            handler: context => {
+                var discarded = context.event.params[3];
+                discarded.controller.moveCard(discarded, 'dead pile');
+                this.game.addMessage('{0} uses {1} to place {2} in {3}\'s dead pile.', this.controller, this, discarded, discarded.controller);
```
Comment (cryogen) [inline server/game/cards/characters/02/sergregorclegane.js]: This is super pedantic but I don't think we typically end sentences in chat with '.'s. We should be consistent either way

File: server/game/cards/characters/02/sergregorclegane.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            }
+        });
+    }
+
+    cardCondition(discarded, card) {
+        return card.location === 'play area' && card.getType() === 'character' && card.getCost() === discarded.getCost();
+    }
+
+    onCardSelected(player, card) {
+        this.game.addMessage('{0} uses {1} to kill {2}.', this.controller, this, card);
```
Comment (cryogen) [inline server/game/cards/characters/02/sergregorclegane.js]: as above

File: server/game/cards/characters/02/sergregorclegane.js (hunk 1 above)
Comment (DukeTax) [inline server/game/cards/characters/02/sergregorclegane.js]: That reminds me, I might want to go through all the messages soon for cleanup. A lot of the older messages are... different.

Comment (ystros) [inline server/game/cards/characters/02/sergregorclegane.js]: Fixed.

File: server/game/cards/characters/02/sergregorclegane.js (hunk 2 above)
Comment (ystros) [inline server/game/cards/characters/02/sergregorclegane.js]: Fixed.

**Pull request merged**